src/components/video-embed/video-embed.tsx

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable react/destructuring-assignment */
+import React from 'react';
+import { CardMedia, CardMediaProps, Box } from '@mui/material';
+import { styled } from '@mui/material/styles';
+
+interface VideoEmbedProps extends CardMediaProps {
+  allow?: string;
+  height?: string;
+  width?: string;
+  component?: string;
+}
+
+const StyledContainer = styled(Box, {
+  name: 'WmeVideoEmbed',
+  slot: 'Root',
+})(() => ({
+  overflow: 'hidden',
+  paddingBottom: '56.25%',
+  position: 'relative',
+  height: 0,
+}));
+
+const StyledMedia = styled(CardMedia, {
+  name: 'WmeVideoEmbed',
+  slot: 'Root',
```

Comment:
```suggestion
  slot: 'Media',
```
This change will give the iframe a unique className, and (I think) provide unique property under the `WmeVideoEmbed` object if a developer wanted to override these styles in the `theme`.